control-panel/build.gradle

```diff
@@ -38,22 +38,22 @@ dependencies {
 }
 
 jar {
-    manifest {
-        name = "${project.parent.name}"
-        symbolicName = "${project.parent.group}.${project.parent.name}"
-        instruction 'Alfresco-Dynamic-Extension', 'true'
-        instruction 'Bundle-Description', 'Control Panel'
-        instruction 'Alfresco-Spring-Configuration', 'com.github.dynamicextensionsalfresco.controlpanel'
-        instruction 'Spring-Context', ';publish-context:=false'
-        instruction 'Import-Package', """*,
-							net.sf.cglib.core,
-							net.sf.cglib.proxy,
-							net.sf.cglib.reflect,
-							org.aopalliance.aop,
-							org.aopalliance.intercept,
-							org.springframework.aop,
-							org.springframework.aop.framework"""
-    }
+    bnd(
+            'Bundle-Name': "${project.name}",
```

Comment:
This drops `parent`, is that intended?

Reply:
Yes it is, this is a bug introduced in a previous PR